src/styles/RecipeFinder.css

```diff
@@ -0,0 +1,82 @@
+.dish-input {
+  padding: 0.2rem;
+  border: 0;
+  font-size: 1.3rem;
+  &:focus {
+    border: 0
+  }
+}
+
+.dish-input:focus-visible {
+  outline: none;
+}
+
+.find-recipe-btn {
+  font-size: 1.2rem;
+  border-radius: 0.75rem;
+  cursor: pointer;
+  background-color: rgb(86, 155, 59);
+  color: white;
+  border: 0.1rem solid rgb(214, 209, 209);
+  padding: 0.4rem;
+  padding-left: 0.7rem;
+  padding-right: 0.7rem;
+
+  font-family: "Inter", sans-serif;
```

Comment:
This was already set in the `body`. Do you need this for the inner input text? Is that why you did it?